go.mod

```diff
@@ -6,13 +6,15 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v0.1.0
 	github.com/go-openapi/spec v0.19.3
+	github.com/go-openapi/validate v0.18.0 // indirect
 	github.com/gobuffalo/packr/v2 v2.7.1
 	github.com/google/go-cmp v0.3.1
 	github.com/google/uuid v1.1.1
 	github.com/infinispan/infinispan-operator v0.0.0-20191028165532-917f400bee62
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/openshift/api v0.0.0-00010101000000-000000000000
-	github.com/openshift/client-go v0.0.0-00010101000000-000000000000
+	github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible
```

Comment:
@xiezhang7 could you confirm if this is the right version of the openshift API? I understand that we should replace it with 4.x.

Reply:
I don't think the version here matters, because of the `replace` at line 35.

Reply:
It's the same major version. And I think we need to support 3.x since we're still running some stories in this OCP version.